### ISSUE TITLE
Bar for contributions?

### ISSUE THREAD

**Issue by kcalvinalvin** [bitcointranscripts/bitcointranscripts#8]
I am wondering what's the bar for contributing transcripts here. Do they have to be purely transcripts or is it ok to have additional notes/comments or even make edits to the actual transcript for clarity.

We have weekly calls for github.com/mit-dci/utreexo and I'm curious if those are deemed fit here. The stuff that we'd talk about is pretty utreexo-specific so I'm sure it wouldn't make sense to most if I were to just make a transcript.

**Comment by adamjonas** [comment]
It's a good question and one I don't have an exact answer to. I think a reasonable line in the sand would be for it to able to stand on its own.

On the other hand, adding edits for additional context is highly encouraged though. Things get stale and links are sparse so feel free to update existing transcripts as you see fit.

**Comment by kanzure** [comment]
utreexo stuff is definitely on-topic and should be included

**Comment by adamjonas** [comment]
I agree that utreexo as a topic is certainly appropriate. My comment was about the inclusion of weekly standup notes/comments in this repo.

**Comment by kcalvinalvin** [comment]
I think I could clarify my original question.

1: Is Utreexo stuff cool (from the answers, that's a definite yes)

2: Does the PRs have to be _purely_ transcripts? Can there be edits to the transcripts to clarify what the speaker is saying? Can there be additional context added (as some topics discussed are hard to follow without context)?

**Comment by michaelfolkson** [comment]
@kcalvinalvin:

I'll add my two cents. I can't speak for anyone else on this thread.

> Is Utreexo stuff cool (from the answers, that's a definite yes)

A definite yes ;)

> Does the PRs have to be purely transcripts?

I feel that an interesting explanatory Reddit post or Bitcointalk post on a technical topic can squeeze into the transcripts definition (others may disagree) as a standalone transcript even though it was never delivered verbally. Especially if you would seek to link to it from another transcript or it would likely get lost and forgotten about otherwise. Just be clear where the "transcript" has come from.

> Can there be edits to the transcripts to clarify what the speaker is saying?

I personally avoided this as it is then subject to my own biases and misunderstandings. What @kanzure has done (and I followed) was to add links to other resources for the interested reader. Either at the beginning of the transcript or at specific points in the transcript. If there is an obvious technical error (which doesn't happen that much at least on the transcripts I have chosen to do personally) I have put the correction in brackets making it clear that it wasn't what the speaker said. But generally I think speakers would prefer their actual words be transcribed accurately (grammar, repeated words, ummms cleanups etc are fine/encouraged to do as the transcriber) as a matter of historical record.

> Can there be additional context added (as some topics discussed are hard to follow without context)?

I would stick to adding links rather than merging talks or different resources together. I think it then becomes a curated article or a topics page (e.g Bitcoin Optech topics [page](https://bitcoinops.org/en/topics/)). The vast majority (if not all) of the words should be attributed to the speaker and as accurately as possible rather than curated by the transcriber.

**Comment by adamjonas** [comment]
It's been 18 months and I think we've got the hang of things. Closing.
